src/app/stores/presence-status-store.ts

```diff
@@ -51,45 +51,45 @@ export class PresenceStatus {
     const os = isWindowsOS ? 'windows' : isMac ? 'macOS' : 'linux';
     const theme = nativeTheme.shouldUseDarkColors ? 'light' : 'dark';
     const assetsPath = `src/renderer/assets/presence-status/${os}/${theme}`;
-
+    let fileExtension = 'png';
+    let iconPlace = '';
+    switch (place) {
+      case 'tray':
+        iconPlace = '-tray';
+        fileExtension = isWindowsOS ? 'ico' : isMac ? 'png' : 'png';
+        break;
+      case 'thumbnail':
+        iconPlace = '-thumbnail';
+        fileExtension = 'ico';
+        break;
+      default:
```

Comment:
By default iconplace's going to be 'taskbar' but intended to be left empty, am I correct?!!